bin/entrypoint.sh

```diff
@@ -38,5 +38,9 @@ unset datestamp location basename
 >&2 bundle install
 
 # Run the command configured for the docker compose service
-# in the docker-compose.yaml file.
-exec bundle exec $@
+# in the docker-compose.yaml file, or a default if none is provided.
+if [ -z "$@" ]; then
+  exec bundle exec thin -R config.ru -p 3000 start
```

Comment:
Consider adding a comment to explain the purpose of the default command added to the entrypoint script. This will help future maintainers understand the rationale behind the change. [medium]

Reply:
Thanks. I added a note to see the full explanation added in the Dockerfile.